<?xml version="1.0" encoding="utf-8"?><testsuites name="pytest tests"><testsuite name="AppiumMobileTests" errors="0" failures="1" skipped="0" tests="8" time="131.415" timestamp="2025-09-17T08:34:38.960198-06:00" hostname="GT_FAB_L31"><testcase classname="test_appium_suite.TestSimpleFlow" name="test_01_click_registrarme" time="21.727"><system-out>--------------------------------- Captured Log ---------------------------------
#x1B[32mINFO    #x1B[0m conftest:conftest.py:156 ============================================================
#x1B[32mINFO    #x1B[0m conftest:conftest.py:157 #x1F680 CONFIGURANDO ENTORNO DE TESTING (UNA SOLA VEZ POR SESIÓN)
#x1B[32mINFO    #x1B[0m conftest:conftest.py:165 ✅ Appium server está corriendo en http://127.0.0.1:4723
#x1B[32mINFO    #x1B[0m conftest:conftest.py:123 Verificando que el dispositivo 'emulator-5554' esté listo y autorizado...
#x1B[32mINFO    #x1B[0m conftest:conftest.py:136 ✅ Dispositivo 'emulator-5554' está en línea y autorizado.
#x1B[32mINFO    #x1B[0m conftest:conftest.py:174 ✅ Entorno configurado correctamente.
#x1B[32mINFO    #x1B[0m conftest:conftest.py:175 ============================================================
#x1B[32mINFO    #x1B[0m conftest:conftest.py:186 #x1F680 Iniciando driver de Appium para TODA LA SESIÓN DE PRUEBAS...
#x1B[32mINFO    #x1B[0m conftest:conftest.py:203 ✅ Driver iniciado exitosamente. La app se mantendrá abierta.
#x1B[32mINFO    #x1B[0m conftest:conftest.py:67 #x1F3A5 Iniciando grabación de video: /sdcard/test_01_click_registrarme_20250917_083451.mp4
#x1B[32mINFO    #x1B[0m conftest:conftest.py:231 #x1F3A5 Grabación iniciada para el test: test_01_click_registrarme
#x1B[32mINFO    #x1B[0m conftest:conftest.py:92 ✅ Video descargado: pytest_videos\test_01_click_registrarme_20250917_083451.mp4
#x1B[32mINFO    #x1B[0m conftest:conftest.py:239 ✅ Video guardado: pytest_videos\test_01_click_registrarme_20250917_083451.mp4
--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="test_appium_suite.TestSimpleFlow" name="test_02_go_back_with_phone_button" time="15.371"><system-out>--------------------------------- Captured Log ---------------------------------
#x1B[32mINFO    #x1B[0m conftest:conftest.py:67 #x1F3A5 Iniciando grabación de video: /sdcard/test_02_go_back_with_phone_button_20250917_083500.mp4
#x1B[32mINFO    #x1B[0m conftest:conftest.py:231 #x1F3A5 Grabación iniciada para el test: test_02_go_back_with_phone_button
#x1B[32mINFO    #x1B[0m conftest:conftest.py:92 ✅ Video descargado: pytest_videos\test_02_go_back_with_phone_button_20250917_083500.mp4
#x1B[32mINFO    #x1B[0m conftest:conftest.py:239 ✅ Video guardado: pytest_videos\test_02_go_back_with_phone_button_20250917_083500.mp4
--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="test_appium_suite.TestSimpleFlow" name="test_03_click_iniciar_sesion" time="7.160"><system-out>--------------------------------- Captured Log ---------------------------------
#x1B[32mINFO    #x1B[0m conftest:conftest.py:67 #x1F3A5 Iniciando grabación de video: /sdcard/test_03_click_iniciar_sesion_20250917_083516.mp4
#x1B[32mINFO    #x1B[0m conftest:conftest.py:231 #x1F3A5 Grabación iniciada para el test: test_03_click_iniciar_sesion
#x1B[32mINFO    #x1B[0m conftest:conftest.py:92 ✅ Video descargado: pytest_videos\test_03_click_iniciar_sesion_20250917_083516.mp4
#x1B[32mINFO    #x1B[0m conftest:conftest.py:239 ✅ Video guardado: pytest_videos\test_03_click_iniciar_sesion_20250917_083516.mp4
--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="test_appium_suite.TestSimpleFlow" name="test_04_escribir_email_y_continuar" time="27.967"><system-out>--------------------------------- Captured Log ---------------------------------
#x1B[32mINFO    #x1B[0m conftest:conftest.py:67 #x1F3A5 Iniciando grabación de video: /sdcard/test_04_escribir_email_y_continuar_20250917_083523.mp4
#x1B[32mINFO    #x1B[0m conftest:conftest.py:231 #x1F3A5 Grabación iniciada para el test: test_04_escribir_email_y_continuar
#x1B[32mINFO    #x1B[0m conftest:conftest.py:92 ✅ Video descargado: pytest_videos\test_04_escribir_email_y_continuar_20250917_083523.mp4
#x1B[32mINFO    #x1B[0m conftest:conftest.py:239 ✅ Video guardado: pytest_videos\test_04_escribir_email_y_continuar_20250917_083523.mp4
--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="test_appium_suite.TestSimpleFlow" name="test_05_click_usuario_y_contrasena" time="7.287"><system-out>--------------------------------- Captured Log ---------------------------------
#x1B[32mINFO    #x1B[0m conftest:conftest.py:67 #x1F3A5 Iniciando grabación de video: /sdcard/test_05_click_usuario_y_contrasena_20250917_083551.mp4
#x1B[32mINFO    #x1B[0m conftest:conftest.py:231 #x1F3A5 Grabación iniciada para el test: test_05_click_usuario_y_contrasena
#x1B[32mINFO    #x1B[0m conftest:conftest.py:92 ✅ Video descargado: pytest_videos\test_05_click_usuario_y_contrasena_20250917_083551.mp4
#x1B[32mINFO    #x1B[0m conftest:conftest.py:239 ✅ Video guardado: pytest_videos\test_05_click_usuario_y_contrasena_20250917_083551.mp4
--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="test_appium_suite.TestSimpleFlow" name="test_06_escribir_usuario_y_contrasena" time="27.787"><system-out>--------------------------------- Captured Log ---------------------------------
#x1B[32mINFO    #x1B[0m conftest:conftest.py:67 #x1F3A5 Iniciando grabación de video: /sdcard/test_06_escribir_usuario_y_contrasena_20250917_083558.mp4
#x1B[32mINFO    #x1B[0m conftest:conftest.py:231 #x1F3A5 Grabación iniciada para el test: test_06_escribir_usuario_y_contrasena
#x1B[32mINFO    #x1B[0m conftest:conftest.py:92 ✅ Video descargado: pytest_videos\test_06_escribir_usuario_y_contrasena_20250917_083558.mp4
#x1B[32mINFO    #x1B[0m conftest:conftest.py:239 ✅ Video guardado: pytest_videos\test_06_escribir_usuario_y_contrasena_20250917_083558.mp4
--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="test_appium_suite.TestSimpleFlow" name="test_07_flujo_completo_productos_y_salir" time="18.264"><failure message="Failed: TEST 7 FALLÓ: No se pudo encontrar el botón 'Ver productos'&#10;assert None is not None">test_appium_suite.py:505: in test_07_flujo_completo_productos_y_salir
    assert ver_productos_button is not None, "No se pudo encontrar el botón 'Ver productos'"
E   AssertionError: No se pudo encontrar el botón 'Ver productos'
E   assert None is not None

During handling of the above exception, another exception occurred:
test_appium_suite.py:642: in test_07_flujo_completo_productos_y_salir
    pytest.fail(f"TEST 7 FALLÓ: {e}")
E   Failed: TEST 7 FALLÓ: No se pudo encontrar el botón 'Ver productos'
E   assert None is not None</failure><system-out>--------------------------------- Captured Log ---------------------------------
#x1B[32mINFO    #x1B[0m conftest:conftest.py:67 #x1F3A5 Iniciando grabación de video: /sdcard/test_07_flujo_completo_productos_y_salir_20250917_083626.mp4
#x1B[32mINFO    #x1B[0m conftest:conftest.py:231 #x1F3A5 Grabación iniciada para el test: test_07_flujo_completo_productos_y_salir
#x1B[32mINFO    #x1B[0m conftest:conftest.py:92 ✅ Video descargado: pytest_videos\test_07_flujo_completo_productos_y_salir_20250917_083626.mp4
#x1B[32mINFO    #x1B[0m conftest:conftest.py:239 ✅ Video guardado: pytest_videos\test_07_flujo_completo_productos_y_salir_20250917_083626.mp4
--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="test_appium_suite.TestSimpleFlow" name="test_debug_current_screen" time="5.637"><system-out>--------------------------------- Captured Log ---------------------------------
#x1B[32mINFO    #x1B[0m conftest:conftest.py:67 #x1F3A5 Iniciando grabación de video: /sdcard/test_debug_current_screen_20250917_083644.mp4
#x1B[32mINFO    #x1B[0m conftest:conftest.py:231 #x1F3A5 Grabación iniciada para el test: test_debug_current_screen
#x1B[32mINFO    #x1B[0m conftest:conftest.py:92 ✅ Video descargado: pytest_videos\test_debug_current_screen_20250917_083644.mp4
#x1B[32mINFO    #x1B[0m conftest:conftest.py:239 ✅ Video guardado: pytest_videos\test_debug_current_screen_20250917_083644.mp4
#x1B[32mINFO    #x1B[0m conftest:conftest.py:209 #x1F3C1 Driver cerrado correctamente al final de la sesión.
#x1B[32mINFO    #x1B[0m conftest:conftest.py:177 ============================================================
#x1B[32mINFO    #x1B[0m conftest:conftest.py:178 #x1F9F9 LIMPIEZA FINAL DEL ENTORNO DE TESTING
#x1B[32mINFO    #x1B[0m conftest:conftest.py:179 ============================================================
--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase></testsuite></testsuites>